d3a_api_client/rest_market.py

```diff
@@ -12,23 +12,19 @@
 
 class RestMarketClient(RestCommunicationMixin):
 
-    def __init__(self, simulation_id, device_id, domain_name,
+    def __init__(self, simulation_id, area_id, domain_name,
```

Comment:
Please notify Colin for this change in order to be able to adapt his scripts accordingly.

Reply:
Will do thanks!